venues/ICLR.cc/2019/Conference/webfield/areachairWebfieldBiddingEnabled.js

```diff
@@ -235,17 +235,16 @@ var renderHeader = function() {
 
 var renderStatusTable = function(profiles, notes, completedReviews, metaReviews, reviewerIds, container) {
   var rows = _.map(notes, function(note) {
-    var revIds = reviewerIds[note.number];
+    var revIds = reviewerIds[note.number] || Object.create(null);
```

Comment:
What does it mean ```Object.create(null)```?

Reply:
You can use just ```{}```

Reply:
This code is copied into this file from https://github.com/iesl/openreview-scripts/pull/779 .
Maybe @zbialecki can better comment on this

Reply:
We should try to avoid the duplication of code. I understand that you need to keep different versions of the webfield, one per stage. The downside is that is difficult to maintain.

Reply:
I agree with Melisa, maintaining 2 versions of this file that we have to keep in sync is not a long term solution.

Object.create(null) is used to create an map that doesn't have all the builtin properties of an object. So using `prop in object` is safe, and calling `.hasOwnProperty()` is not necessary. So it's better if you're going to be using `for...in` loops.

Reply:
I imagined that. Thanks for the explanation. 

Webfields are becoming too complex, we need to work on this after ICLR.